easybuild/easyconfigs/a/alleleCount/alleleCount-4.2.1_make.patch

```diff
@@ -0,0 +1,70 @@
+Do not build local HTSlib, Use EasyBuild defined CPPFLAGS. Clean up Makefile variables
+--- alleleCount-4.2.1.orig/c/Makefile	2020-11-17 08:02:34.000000000 -0800
++++ alleleCount-4.2.1/c/Makefile	2022-07-19 15:53:08.141102000 -0700
+@@ -1,30 +1,15 @@
+ VERSION=$(shell perl -I../perl/lib -MSanger::CGP::AlleleCount -e 'print Sanger::CGP::AlleleCount->VERSION;')
+ 
+ #Compiler
+-CC = gcc -O3 -DALLELECOUNTER_VERSION='"$(VERSION)"'
++CC ?= gcc
+ 
+ #compiler flags
+ # -g adds debug info to the executable file
+ # -Wall turns on most warnings from compiler
+-CFLAGS = -g -Wall
+-
+-#Location of samtools/htslib libraries
+-HTSLOC?=$(HTSLIB)
+-
+-HTSTMP?=./htslib_tmp
+-prefix?=/usr/local/
++CFLAGS = -O3 -Wall -DALLELECOUNTER_VERSION='"$(VERSION)"'
```

Comment:
@fizwit could you explain this part please?

Reply:
Removed most references to HTSlib to allow linking with EB HTSlib.
I guess that prefix line could stay? It will only be set if prefix is not defined.
Not sure what I am doing with CFLAGS, what would work better for CFLAGS?